static/app/views/performance/table.tsx

```diff
@@ -475,4 +478,10 @@ function Table(props: Omit<Props, 'summaryConditions'> & {summaryConditions?: st
   return <_Table {...props} summaryConditions={summaryConditions} />;
 }
 
+// Align the contained IconStar with the IconStar buttons in individual table
+// rows, which have 2px padding + 1px border.
+const TeamKeyTransactionWrapper = styled('div')`
+  padding: 3px;
+`;
+
```

Comment:
We can maybe ask @vuluongj20 about whether the button having a padding of 1-2px even with the settings to make it borderless makes sense, in which case, yeah we need to make our use of the icon have the same padding so they match.

Reply:
Hmm lemme check to see why we have that 2px padding – I can't remember the reason for it